files/image_config/process-reboot-cause/process-reboot-cause

```diff
@@ -47,6 +51,16 @@ def log_error(msg):
 
 
 # ============================= Functions =============================
+def is_warmfast_reboot_from_proc_cmdline():
+    if os.path.isfile(REBOOT_CAUSE_KEXEC):
+        cause_file = open(REBOOT_CAUSE_KEXEC, "r")
```

Comment:
files was not closed, try to use ```with open(REBOOT_CASE_KEXEC, "r"):```

Reply:
fixed. use "with" for all open file cases.